Library/Formula/bde.rb

```diff
@@ -0,0 +1,25 @@
+class Bde < Formula
+  desc "Basic Development Environment: foundational C++ libraries used at Bloomberg"
+  homepage "https://github.com/bloomberg/bde"
+  url "https://github.com/bloomberg/bde/archive/BDE_3.0.0.0.tar.gz"
+  sha256 "c6f295947c1af5f0d4e728e4d6801c4b29bb35a742faebc058f86b36722e8cdd"
+
+  resource "bde-tools" do
+    url "https://github.com/bloomberg/bde-tools/archive/281b4317f4fe631588e808d7dc6a2b81b97a063d.tar.gz"
+    sha256 "42c32c9f78b7183b3c9cf875ff0fbdb1a437dcd93e17fa1cb4632f0887ecc138"
+  end
+
+  def install
+    Pathname.pwd.install resource("bde-tools")
+
+    system "python", "./bin/waf", "configure", "--prefix=#{prefix}"
+    system "python", "./bin/waf", "build"
+    system "python", "./bin/waf", "install"
+  end
+
+  test do
+    # bde tests are incredibly performance intensive, just skip for now
```

Comment:
We'll need some form of tests. We don't accept new formulae that can't be tested because it bites us _so_ often.

See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!